### ISSUE TITLE
Support OTF and FNT

### ISSUE THREAD

**Issue by DasNaCl** [UnderwaterApps/overlap2d#72]
Currently TTF only seems to be the font-way of fonting here. But more formats should be supported, especially OTF and sometimes I (and probably you, too) already have a font as a bitmap font, so better support that stuff, too.

**Comment by azakhary** [comment]
Makes sense for sure, as new version is pushed, would be awesome if someone can contribute this. But if not, I'll take a look.
